docs/model-dev-guide/dtrain/dtrain-introduction.rst

```diff
@@ -8,93 +8,93 @@
+-  Use custom optimizers designed for large batch training, such as `RAdam
    <https://github.com/LiyuanLucasLiu/RAdam>`_, `LARS <https://arxiv.org/pdf/1708.03888.pdf>`_, or
-   `LAMB <https://arxiv.org/pdf/1904.00962.pdf>`_. We have found RAdam especially effective.
+   `LAMB <https://arxiv.org/pdf/1904.00962.pdf>`_. In our experience, RAdam has been particularly
+   effective.
 
-These techniques often require hyperparameter modifications. To automate this process, we encourage
-users to utilize the :ref:`hyperparameter-tuning` capabilities in Determined.
+Applying these techniques often requires hyperparameter modifications. To help automate this
+process, use the :ref:`hyperparameter-tuning` capabilities in Determined.
 
 ***********************************************
  Model Characteristics that Affect Performance
 ***********************************************
 
-Deep learning models typically perform dense updates, where every model parameter is updated for
-every training sample. Because of this, the amount of communication per mini-batch (step 3 in the
-distributed training loop) is dependent on the number of model parameters. Models that have fewer
-parameters such as `ResNet-50 <https://arxiv.org/pdf/1512.03385.pdf>`_ (~30 million parameters)
-train more efficiently in distributed settings than models with more parameters such as `VGG-16
-<https://arxiv.org/pdf/1505.06798.pdf>`_ (~136 million parameters). If planning to utilize
-distributed training, we encourage users to be mindful of their model size when designing models.
-
-*******************************
- Debug Performance Bottlenecks
-*******************************
-
-When scaling up distributed training, it's fairly common to see non-linear speedup when scaling from
-one machine to two machines as intra-machine communication (e.g., NVLink) is often significantly
-faster than inter-machine communication. Scaling up beyond two machines often provides close to
-linear speed-up, but it does vary depending on the model characteristics. If observing unexpected
-scaling performance, assuming you have scaled your ``global_batch_size`` proportionally with
-``slots_per_trial``, it's possible that training performance is being bottlenecked by network
-communication or disk I/O.
+Deep learning models typically perform dense updates, meaning every model parameter is updated for
+every training sample. Consequently, the quantity of communication per mini-batch (step 3 in the
+distributed training loop) is dependent on the number of model parameters. Models having fewer
+parameters like `ResNet-50 <https://arxiv.org/pdf/1512.03385.pdf>`_ (~30 million parameters) train
+more efficiently in distributed settings than models with more parameters such as `VGG-16
+<https://arxiv.org/pdf/1505.06798.pdf>`_ (~136 million parameters). If you are planning on utilizing
+distributed training, consider the size of your model when designing it.
```

Comment:
Not super-helpful advice, and given the title of the section I would also expect an explanation of multiple characteristics that affect performance, rather than just one.  IMO we could just cut this paragraph/section.

Reply:
ok, removed